transports/janus_mqtt.c

```diff
@@ -157,27 +162,95 @@ int janus_mqtt_client_message_arrived(void *context, char *topicName, int topicL
 void janus_mqtt_client_delivery_complete(void *context, MQTTAsync_token token);
 int janus_mqtt_client_connect(janus_mqtt_context *ctx);
 void janus_mqtt_client_connect_success(void *context, MQTTAsync_successData *response);
+#ifdef MQTTVERSION_5
+	void janus_mqtt_client_connect_success5(void *context, MQTTAsync_successData5 *response);
```

Comment:
No need to indent here, it's a function declaration as the others; same for the others below.
Thinking about it, it might make sense to group all the v5 declarations together in a single `#ifdef`? Would make everything more readable IMO.